Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm

```diff
@@ -97,6 +97,7 @@
 #import <WebCore/ThemeMac.h>
 #import <WebCore/VisibleUnits.h>
 #import <WebCore/WindowsKeyboardCodes.h>
+#import <pal/spi/cg/CoreGraphicsSPI.h>
```

Comment:
Why are we adding this import? It's unrelated to the stated purpose of the change.